runner_manager/logging.py

```diff
@@ -0,0 +1,20 @@
+import logging
+
+from runner_manager.models.settings import Settings
+
+settings = Settings()
+
+log_level = getattr(logging, settings.log_level.upper(), logging.INFO)
```

Comment:
Check exactly what is the type of logging.INFO and all the others,
You may be able to do a specific type definitions in settings allowing you to remove the need of applying the transformation here.

Or apply this transformation directly in settings with the Field function

Reply:
logging.INFO is a int.

Reply:
I'm not sure if this is what you want but i modified it
`log_level: str = Field(default="INFO", pattern=r'^[A-Z]+$')`